.github/workflows/backend_build_check.yml

```diff
@@ -0,0 +1,24 @@
+name: build database-service backend
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        os: [ubuntu-lastest]
+        node-version: [12.x] 
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: npm install
+      run: |
+        cd database-service; npm install --dev
```

Comment:
You can use `working-directory` keyword on step or job level to define the working directory for the shell commands. With `defaults.run.working-directory` you can set it as default for all the steps. https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#defaultsrun

Reply:
very nice, i didn't know about this one, but  i like it.